dss/blobstore/s3.py

```diff
@@ -186,3 +186,46 @@ def copy(
                 multipart_chunksize=64 * 1024 * 1024,
             ),
         )
+
+    def find_next_missing_parts(
+            self,
+            bucket: str,
+            key: str,
+            upload_id: str,
+            part_count: int,
+            search_start: int=1,
+            return_count: int=1) -> typing.Sequence[int]:
+        """
+        Given a `bucket`, `key`, and `upload_id`, find the next N missing parts of a multipart upload, where
+        N=`return_count`.  If `start_search` is provided, start the search at part M, where M=`search_start`.
```

Comment:
s/start_search/search_start/

Also, `part_count` and `return_count` are not documented, is that intentional?

Also, if the function gets called with part_count < search_start (and there are not enough missing parts), it will loop forever. Suggest adding an assertion that search_start <= part_count.